pkg/controllers/work/apply_controller.go

```diff
@@ -50,6 +52,8 @@ import (
 
 const (
 	workFieldManagerName = "work-api-agent"
+	fleetSystemNamespace = "fleet-system"
+	configMapNamePrefix  = "configmap-"
```

Comment:
Do we need pkg level constants / vars if they are only used once within the source? I believe I've received that feedback before.